ruby/ruby/IceBox.rb

```diff
@@ -0,0 +1,5 @@
+#
+# Copyright (c) ZeroC, Inc. All rights reserved.
+#
+
+require 'IceBox/IceBox'
```

Comment:
Here we want to include `IceBox/ServiceManager.rb` this file was renamed in 3.8.